server/utils/utils.go

```diff
@@ -55,3 +55,11 @@ func Map(vs []string, f func(string) string) []string {
 	}
 	return vsm
 }
+
+func IsJiraCloudURL(jiraURL string) (bool, error) {
+	u, err := url.Parse(jiraURL)
+	if err != nil {
+		return false, err
+	}
+	return strings.Contains(u.Hostname(), ".atlassian.net"), nil
```

Comment:
1/5 maybe, `...HasSuffix` is more appropriate?

Reply:
You are probably right. I used HasSuffix at first but it doesn't catch when you accidentally write something after ".net". Maybe we should sanity check the domain name, but that's a different issue.